_episodes/03-github-pages.md

```diff
@@ -3,275 +3,126 @@ title: "Hosting Pages and Websites on GitHub"
+{: .callout}
 
-What happened behind the hood, is that GitHub converted markdown to HTML. At the top of your website, the name of your repository appears as a link to the project homepage.
+Either of the above two approaches to turning a repository to a website will give you the same result - the `gh-pages` approach is perhaps more common as it favours convention over configuration.
 
-> ## What if my browser does not open?
->
-> If your default web browser did not open and does not show the project website, we recommend to manually open your favourite web browser and paste the project web address.
->
-{: .callout}
+## Understanding GitHub Pages URLs
 
-### Understanding GitHub web addresses
+You may have noticed a slightly strange URL for your website appearing in that green box with a "tick" in front of it.
+This URL was generated by GitHub Pages and is not random. It is formatted as 
+'https://GITHUB_USERNAME.github.io/REPOSITORY_NAME' and is formed by appending:
 
-The resulting website does not have a random web address. It is formed by:
+- your GitHub username or organisation name under which the repository is created (GITHUB_USERNAME)
+- '.github.io/' (GitHub's web hosting domain)
+- the repository name (REPOSITORY_NAME)
 
-- your github username or organization name
-- followed by `github.io`
-- And finally the repository name: / {{ site.project-pages }}
+Because the repository name is unique within one's personal or organisational GitHub account - this naming 
+convention gives us a way
+of neatly creating Web address for just about any GitHub repository without any conflicts.
 
-> ## Customize domain
+> ## Customising domain
 > **TODO**
 {: .callout}
 
-### Update repository details
+## Updating Repository Details With Website URL
+
+On the right hand side of your repository on GitHub, you can see the details of your repository under 'About'. It is good
+practice to update your repository details with a brief description. This is also a place where you can put 
+your repository's Web URL (as not everyone will have access to your repository's `Settings` to find it).
 
-On the right hand side of your repository on GitHub, you can edit the details of your repository (click on the wheel as shown on the figure below) and add the link to your website. This would help anyone to access it when visiting your github repository.
+You can edit the details of your repository by clicking on the little cog button as shown on the figure below.
 
 ![Repository details](../fig/repository_details.png)
 
-> ## Add new content to your website
->
-> Add a new section **About** in `index.md`. You can edit `index.md` from the GitHub interface and directly commit your changes.
-> Then view your changes on the website.
+By doing this, you add a link to the repository's website on your repository's landing page and anyone (including yourself) can access it quickly when visiting your GitHub repository.
+
```

Comment:
```suggestion

Up to now, the content of your webpage is identical to what visitors to your repository on GitHub will see in the project's `README.md` file. It is often better to have different content in the homepage of your site - aimed at visitors to the website - than in the README, which is supposed to provide information about the GitHub repository e.g. license information, local installation instructions, the structure and configuration of the repository, etc. By default, the homepage for a GitHub Pages website is built from a file called `index.md`: in the absence of a file with that name the "fallback" option is to use `README.md`, which is why your homepage is currently being built from that file.

To separate the contents of the repository's README from the website's homepage, create a new file called `index.md` and copy the contents of the README into that file. We'll add more content to that file in the next exercise.
```

Proposing this to bridge the gap between README and `index.md` for the homepage. Feel free to overwrite/ignore any of it!